cmd/insights/root.go

```diff
@@ -68,20 +70,35 @@ func exitWithError(message string, err error) {
 	}
 }
 
+// insightsAPINotRequired returns true if Insights API connectivity is not
+// required for the supplied Cobra.command.
+func insightsAPINotRequired(cmd *cobra.Command) bool {
+	return cmd.Use == "help [command]" ||
+		cmd.Use == "insights-cli" ||
+		cmd.Use == "validate" ||
+		(cmd.Parent().Use == "validate" && cmd.Use == "opa")
+}
+
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&logLevel, "log-level", "", logrus.InfoLevel.String(), "Logrus log level.")
 	rootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "./fairwinds-insights.yaml", "Configuration file")
 	rootCmd.PersistentFlags().StringVarP(&organization, "organization", "", "", "Fairwinds Insights Organization name")
+	rootCmd.PersistentFlags().BoolVarP(&noDecoration, "no-decoration", "", false, "Do not include decorative characters in output, such as tree visualization.")
 }
 
 func preRun(cmd *cobra.Command, args []string) {
+	if noDecoration {
+		treeprint.EdgeTypeLink = ""
+		treeprint.EdgeTypeMid = ""
+		treeprint.EdgeTypeEnd = ""
```

Comment:
I think `Mid` and `End` should probably be `"  "` (two or more spaces)

Reply:
I've updated this, here's what the output looks like now, is this what you imagine?

```
$ ./cli list all --no-decoration
.
   opa
       test2 (v2)
   rules
       Assign API Action Items
```

Reply:
Robert [replied looks good to me](https://fairwindsops.slack.com/archives/CNQBGRGFN/p1649255121080529?thread_ts=1649248209.397319&cid=CNQBGRGFN).